.azure/pipelines/ci.yml

```diff
@@ -326,11 +326,43 @@ stages:
             publishFeedCredentials: 'DevDiv - VS package feed'
 
 
-  # Build MacOS
+  # Build MacOS arm64
+  - template: jobs/default-build.yml
+    parameters:
+      jobName: MacOs_arm64_build
+      jobDisplayName: "Build: macOS arm64"
+      agentOs: macOs
+      buildArgs:
+        --arch arm64
+        --pack
+        --all
+        --no-build-nodejs
+        --no-build-java
+        -p:OnlyPackPlatformSpecificPackages=true
+        -p:AssetManifestFileName=aspnetcore-MacOS_arm64.xml
```

Comment:
does this new asset manifest file need to be added, or is it generated by the build some place?

Reply:
This is generated, I've made the same change for linux-musl-arm some time ago.

Reply:
ok cool.